Sources/Corvus/Authentication/CorvusModelUser.swift

```diff
@@ -0,0 +1,5 @@
+import Fluent
+import Vapor
+
+/// A protocol that wraps `Vapor`'s `ModelUser` for consistency in naming.
+public protocol CorvusModelUser: ModelUser {}
```

Comment:
@bmikaili What do you think about adding some protocol requirements here so we can allow users to write their own instances of a `CorvusModelUser` similar to the Token?
As `CorvusUser` is final and has to stay that way you can't simply add a subclass of `CorvusUser`. But if we move most of the implementation from `CorvusUser` here and add them as protocol extensions try to use generic types construed to `CorvusModelUser` throughout the rest of the code we might enable a framework user to easily write his own instance of `CorvusModelUser` that e.g. includes a fist and last name as additional fields and the code still works fine. See the need to implement the key paths in Tests/CorvusTests/Models/CustomUser.swift

Reply:
They can already extend `CorvusModelUser` like it's done with `CustomUser`. I tried giving the protocol some more default implementation, but that won't work because: "Static stored properties not supported in protocol extensions"